packages/legend-application-query/style/_query-builder-explorer.scss

```diff
@@ -474,7 +474,7 @@
   &__node__container--unmapped &__type-icon &__icon,
   &__node__container--unmapped &__node__label__derived-property,
   &__node__container--unmapped &__node__label__multiple {
-    color: var(--color-dark-grey-350);
+    color: var(--color-dark-grey-350) !important;
```

Comment:
generally, we try to avoid `!important` since later, it's hard to override and we would have to make everything __more__ `!important`. So, let's try the followings:
1. re-arrange classes so the rule for `unmapped` goes later (I think even the order of CSS classes matter so consider reordering in the component is also another way)
2. have a more specific rule, so you can say if the node is `selected` and `unmapped`, show text as grey

Reply:
actually, if this is introduced by #1338 where you set `color: var(--color-primitive) !important;`, we should revert that change and find a different strategy.